test/CMakeLists.txt

```diff
@@ -14,11 +14,13 @@ function( add_t8_test )
     # If MPI is enabled, and no custom test command is provided, use mpirun -np 4 as the default command.
     # If MPI is not enabled, use the custom test command as is.
     if ( T8CODE_ENABLE_MPI )
-        if ( T8CODE_CUSTOM_TEST_COMMAND STREQUAL "" )
+        if (${ADD_T8_TEST_NAME} MATCHES "_serial")
+            separate_arguments(T8CODE_TEST_COMMAND_LIST NATIVE_COMMAND "")
+        elseif ( T8CODE_CUSTOM_TEST_COMMAND STREQUAL "" )
             separate_arguments (T8CODE_TEST_COMMAND_LIST NATIVE_COMMAND "mpirun -np 4")
         else ( T8CODE_CUSTOM_TEST_COMMAND STREQUAL "" )
             separate_arguments (T8CODE_TEST_COMMAND_LIST NATIVE_COMMAND ${T8CODE_CUSTOM_TEST_COMMAND})
-        endif ( T8CODE_CUSTOM_TEST_COMMAND STREQUAL "" )
+        endif ()
```

Comment:
This logic does not allow for custom test commands for serial tests. Maybe we can introduce a custom parallel command and a custom serial command